src/Components/AutoUpdate/UpdateModal.scss

```diff
@@ -0,0 +1,48 @@
+.Modal {
+  &.UpdateModal {
+    .Update {
+      .releaseName {
+        width: 100%
+      }
+
+      .releaseNotesOrError {
+        height: 5rem;
+        background-color: var(--app-button-primary-disabled-border-color);
+        width: 100%;
+        margin-bottom: 1rem;
+        border: 1px solid #888888;
+        overflow-y: scroll;
+      }
+
+      .updateDetails {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+      }
+
+      .ProgressBar {
+        width: 200px
+      }
+
+      .downloadSpeed {
+        text-align: right;
```

Comment:
I went back-and-forth on this, and I'm still not thrilled with the layout. As it is, I think it's fine for a first pass, but we might want to polish it up a bit next release.